src/nvim/screen.c

```diff
@@ -3142,7 +3142,6 @@ win_line (
             && (*mb_char2cells)(mb_c) == 2) {
           c = '>';
           mb_c = c;
-          mb_l = 1;
```

Comment:
could do `(void)mb_l;` to avoid the warning instead.

Reply:
sure. As far as my C knowledge permits - it casts variable to void, but the value does not change. Is that a trick for a developer to signal that `mb_l` is irrelevant in this case, but keep code consistent (keep `mb_l` and `mb_c` together?)

Reply:
> Is that a trick for a developer to signal that `mb_l` is irrelevant in this case, but keep code consistent (keep `mb_l` and `mb_c` together?)

Yes, and it is a no-op way of "using" the variable so that analyzers do not flag it.